### ISSUE TITLE
False positive?

### ISSUE THREAD

**Issue by frederikstonge** [crasowas/app_store_required_privacy_manifest_analyser#14]
The script gives me 4 warnings at the end, but if you search for MLKitCommon, it says that the manifest has been found...

```bash
    ~/Development/Company/spark-mobileapp-shop    master !12 ?1  flutter clean                                                       ✔  07:23:57 AM  
Cleaning Xcode workspace...                                         3.8s
Deleting build...                                                  511ms
Deleting .dart_tool...                                               2ms
Deleting Generated.xcconfig...                                       0ms
Deleting flutter_export_environment.sh...                            0ms
Deleting Flutter.podspec...                                          0ms
Deleting .flutter-plugins-dependencies...                            0ms
Deleting .flutter-plugins...                                         0ms
    ~/Dev/Co/spark-mobileapp-shop    master !12 ?1  flutter pub get                                                           ✔  5s   07:38:03 AM  
Resolving dependencies... 
  fast_immutable_collections 8.2.0 (10.2.2 available)
  frontend_server_client 3.2.0 (4.0.0 available)
  go_router 14.0.1 (14.0.2 available)
  intl 0.18.1 (0.19.0 available)
  js 0.6.7 (0.7.1 available)
  leak_tracker 10.0.0 (10.0.5 available)
  leak_tracker_flutter_testing 2.0.1 (3.0.5 available)
  leak_tracker_testing 2.0.1 (3.0.1 available)
  local_auth_ios 1.1.7 (discontinued replaced by local_auth_darwin)
  logger 1.4.0 (2.2.0 available)
  material_color_utilities 0.8.0 (0.11.1 available)
  meta 1.11.0 (1.14.0 available)
  qs_dart 1.0.9 (1.0.10 available)
  test 1.24.9 (1.25.4 available)
  test_api 0.6.1 (0.7.1 available)
  test_core 0.5.9 (0.6.2 available)
  vm_service 13.0.0 (14.2.1 available)
  webview_flutter 3.0.4 (4.7.0 available)
  webview_flutter_android 2.10.4 (3.16.0 available)
  webview_flutter_platform_interface 1.9.5 (2.10.0 available)
  webview_flutter_wkwebview 2.9.5 (3.13.0 available)
Got dependencies!
1 package is discontinued.
20 packages have newer versions incompatible with dependency constraints.
Try `flutter pub outdated` for more information.
⢿⢿^R
    ~/Dev/Co/spark-mobileapp-shop    master !12 ?1  cd ios                                                                   ✔  16s   07:38:22 AM  
    ~/Dev/Co/spark-mobileapp-shop/ios    master !12 ?1  pod install --repo-update                                                    ✔  07:38:24 AM  
Updating local specs repositories
Analyzing dependencies
firebase_analytics: Using Firebase SDK version '10.24.0' defined in 'firebase_core'
firebase_core: Using Firebase SDK version '10.24.0' defined in 'firebase_core'
Run script to upload symbols already exists.
firebase_crashlytics: Using Firebase SDK version '10.24.0' defined in 'firebase_core'
firebase_performance: Using Firebase SDK version '10.24.0' defined in 'firebase_core'
firebase_remote_config: Using Firebase SDK version '10.24.0' defined in 'firebase_core'
Downloading dependencies
Installing CSCrashReporter (1.0.0)
Installing CS_iOS_SDK (4.32.0)
Installing Firebase (10.24.0)
Installing FirebaseABTesting (10.24.0)
Installing FirebaseAnalytics (10.24.0)
Installing FirebaseCore (10.24.0)
Installing FirebaseCoreExtension (10.24.0)
Installing FirebaseCoreInternal (10.24.0)
Installing FirebaseCrashlytics (10.24.0)
Installing FirebaseInstallations (10.24.0)
Installing FirebasePerformance (10.24.0)
Installing FirebaseRemoteConfig (10.24.0)
Installing FirebaseRemoteConfigInterop (10.24.0)
Installing FirebaseSessions (10.24.0)
Installing FirebaseSharedSwift (10.24.0)
Installing Flutter (1.0.0)
Installing GTMSessionFetcher (3.4.1)
Installing GoogleAppMeasurement (10.24.0)
Installing GoogleDataTransport (9.4.1)
Installing GoogleMLKit (6.0.0)
Installing GoogleToolboxForMac (4.2.1)
Installing GoogleUtilities (7.13.0)
Installing GoogleUtilitiesComponents (1.1.0)
Installing MLImage (1.0.0-beta5)
Installing MLKitBarcodeScanning (5.0.0)
Installing MLKitCommon (11.0.0)
Installing MLKitVision (7.0.0)
Installing PromisesObjC (2.4.0)
Installing PromisesSwift (2.4.0)
Installing SwiftProtobuf (1.26.0)
Installing app_settings (5.1.1)
Installing contentsquare (0.1.0)
Installing firebase_analytics (10.10.4)
Installing firebase_core (2.30.1)
Installing firebase_crashlytics (3.5.4)
Installing firebase_performance (0.9.4-4)
Installing firebase_remote_config (4.4.4)
Installing flutter_keyboard_visibility (0.0.1)
Installing flutter_native_splash (0.0.1)
Installing flutter_secure_storage (6.0.0)
Installing geolocator_apple (1.2.0)
Installing haptic_feedback (0.4.2)
Installing local_auth_darwin (0.0.1)
Installing local_auth_ios (0.0.1)
Installing mobile_scanner (5.0.0)
Installing nanopb (2.30910.0)
Installing open_file (0.0.1)
Installing package_info_plus (0.4.5)
Installing path_provider_foundation (0.0.1)
Installing sqflite (0.0.3)
Installing url_launcher_ios (0.0.1)
Installing video_player_avfoundation (0.0.1)
Installing webview_flutter_wkwebview (0.0.1)
Generating Pods project
Integrating client project
Pod installation complete! There are 23 dependencies from the Podfile and 53 total pods installed.

[!] Automatically assigning platform `iOS` with version `12.4` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

[!] CocoaPods did not set the base configuration of your project because your project already has a custom config set. In order for CocoaPods integration to work at all, please either set the base configurations of the target `Runner` to `Target Support Files/Pods-Runner/Pods-Runner.profile.xcconfig` or include the `Target Support Files/Pods-Runner/Pods-Runner.profile.xcconfig` in your build configuration (`Flutter/Release.xcconfig`).
    ~/Dev/Co/spark-mobileapp-shop/ios    master !12 ?1  sh ../.privacy/privacy_manifest_analyser.sh .                         ✔  5s   07:38:34 AM  

==================== Analyzing Target Directory ====================

⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

==================== Analyzing Pods Directory ====================

Analyzing CSCrashReporter ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing CS_ErrorAnalysis ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 2
[0] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/CS_ErrorAnalysis/ContentsquareErrorAnalysisModule.xcframework/ios-arm64/ContentsquareErrorAnalysisModule.framework/ContentsquareErrorAnalysisModule
[1] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/CS_ErrorAnalysis/ContentsquareErrorAnalysisModule.xcframework/ios-arm64_x86_64-simulator/ContentsquareErrorAnalysisModule.framework/ContentsquareErrorAnalysisModule
🛠️  Descriptions for the following required API reason(s) may be missing: 1
[0] NSPrivacyAccessedAPICategoryUserDefaults

Analyzing CS_iOS_SDK ...
💡 Found privacy manifest file(s): 2
[0] ./Pods/CS_iOS_SDK/ContentsquareModule.xcframework/ios-arm64_x86_64-simulator/ContentsquareModule.framework/PrivacyInfo.xcprivacy
[1] ./Pods/CS_iOS_SDK/ContentsquareModule.xcframework/ios-arm64/ContentsquareModule.framework/PrivacyInfo.xcprivacy
API usage analysis result(s): 2
[0] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/CS_iOS_SDK/ContentsquareModule.xcframework/ios-arm64/ContentsquareModule.framework/ContentsquareModule
[1] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/CS_iOS_SDK/ContentsquareModule.xcframework/ios-arm64_x86_64-simulator/ContentsquareModule.framework/ContentsquareModule
✅ All required API reasons have been described in the privacy manifest.

Analyzing FMDB 🎯 ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/FMDB/privacy/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing Firebase ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing FirebaseABTesting 🎯 ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/FirebaseABTesting/FirebaseABTesting/Sources/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing FirebaseAnalytics ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing FirebaseCore 🎯 ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/FirebaseCore/FirebaseCore/Sources/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 3
[0] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/FirebaseCore/FirebaseCore/Sources/FIRAnalyticsConfiguration.m
[1] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/FirebaseCore/FirebaseCore/Sources/FIRApp.m
[2] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/FirebaseCore/FirebaseCore/Sources/FIRLogger.m
✅ All required API reasons have been described in the privacy manifest.

Analyzing FirebaseCoreExtension 🎯 ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/FirebaseCoreExtension/FirebaseCore/Extension/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing FirebaseCoreInternal 🎯 ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/FirebaseCoreInternal/FirebaseCore/Internal/Sources/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 3
[0] NSPrivacyAccessedAPICategoryUserDefaults:UserDefaults:./Pods/FirebaseCoreInternal/FirebaseCore/Internal/Sources/HeartbeatLogging/HeartbeatLoggingTestUtils.swift
[1] NSPrivacyAccessedAPICategoryUserDefaults:UserDefaults:./Pods/FirebaseCoreInternal/FirebaseCore/Internal/Sources/HeartbeatLogging/Storage.swift
[2] NSPrivacyAccessedAPICategoryUserDefaults:UserDefaults:./Pods/FirebaseCoreInternal/FirebaseCore/Internal/Sources/HeartbeatLogging/StorageFactory.swift
✅ All required API reasons have been described in the privacy manifest.

Analyzing FirebaseCrashlytics 🎯 ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/FirebaseCrashlytics/Crashlytics/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 4
[0] NSPrivacyAccessedAPICategoryUserDefaults:UserDefaults,NSUserDefaults:./Pods/FirebaseCrashlytics/Crashlytics/Crashlytics/FIRCLSUserDefaults/FIRCLSUserDefaults.m
[1] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/FirebaseCrashlytics/Crashlytics/Crashlytics/Handlers/FIRCLSException.mm
[2] NSPrivacyAccessedAPICategoryFileTimestamp:stat:./Pods/FirebaseCrashlytics/Crashlytics/Crashlytics/Helpers/FIRCLSFile.m
[3] NSPrivacyAccessedAPICategoryFileTimestamp:stat:./Pods/FirebaseCrashlytics/Crashlytics/Shared/FIRCLSMachO/FIRCLSMachO.m
🛠️  Descriptions for the following required API reason(s) may be missing: 1
[0] NSPrivacyAccessedAPICategoryFileTimestamp

Analyzing FirebaseInstallations 🎯 ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/FirebaseInstallations/FirebaseInstallations/Source/Library/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing FirebaseMessaging 🎯 ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/FirebaseMessaging/FirebaseMessaging/Sources/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing FirebasePerformance ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 3
[0] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/FirebasePerformance/FirebasePerformance/Sources/Configurations/FPRConfigurations+Private.h
[1] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/FirebasePerformance/FirebasePerformance/Sources/Configurations/FPRConfigurations.m
[2] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/FirebasePerformance/FirebasePerformance/Sources/Configurations/FPRRemoteConfigFlags+Private.h
🛠️  Descriptions for the following required API reason(s) may be missing: 1
[0] NSPrivacyAccessedAPICategoryUserDefaults

Analyzing FirebaseRemoteConfig 🎯 ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/FirebaseRemoteConfig/FirebaseRemoteConfig/Swift/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 1
[0] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/FirebaseRemoteConfig/FirebaseRemoteConfig/Sources/RCNUserDefaultsManager.m
✅ All required API reasons have been described in the privacy manifest.

Analyzing FirebaseRemoteConfigInterop ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing FirebaseSessions ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 1
[0] NSPrivacyAccessedAPICategoryUserDefaults:UserDefaults:./Pods/FirebaseSessions/FirebaseSessions/Sources/Settings/SettingsCacheClient.swift
🛠️  Descriptions for the following required API reason(s) may be missing: 1
[0] NSPrivacyAccessedAPICategoryUserDefaults

Analyzing FirebaseSharedSwift ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing GTMSessionFetcher 🎯 ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/GTMSessionFetcher/Sources/Core/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 2
[0] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/GTMSessionFetcher/Sources/Core/GTMSessionFetcher.m
[1] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/GTMSessionFetcher/Sources/Core/Public/GTMSessionFetcher/GTMSessionFetcher.h
✅ All required API reasons have been described in the privacy manifest.

Analyzing GoogleAppMeasurement ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing GoogleDataTransport 🎯 ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/GoogleDataTransport/GoogleDataTransport/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing GoogleMLKit ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing GoogleToolboxForMac 🎯 ...
💡 Found privacy manifest file(s): 2
[0] ./Pods/GoogleToolboxForMac/Resources/Logger/PrivacyInfo.xcprivacy
[1] ./Pods/GoogleToolboxForMac/Resources/Base/PrivacyInfo.xcprivacy
API usage analysis result(s): 2
[0] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/GoogleToolboxForMac/Foundation/GTMLogger.h
[1] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/GoogleToolboxForMac/Foundation/GTMLogger.m
🛠️  Descriptions for the following required API reason(s) may be missing: 1
[0] NSPrivacyAccessedAPICategoryUserDefaults

Analyzing GoogleUtilities 🎯 ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/GoogleUtilities/GoogleUtilities/Privacy/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 4
[0] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/GoogleUtilities/GoogleUtilities/Environment/GULHeartbeatDateStorageUserDefaults.m
[1] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/GoogleUtilities/GoogleUtilities/Environment/Public/GoogleUtilities/GULHeartbeatDateStorageUserDefaults.h
[2] NSPrivacyAccessedAPICategoryFileTimestamp:creationDate,NSURLCreationDateKey:./Pods/GoogleUtilities/GoogleUtilities/Network/GULNetworkURLSession.m
[3] NSPrivacyAccessedAPICategoryUserDefaults:UserDefaults:./Pods/GoogleUtilities/GoogleUtilities/UserDefaults/GULUserDefaults.m
✅ All required API reasons have been described in the privacy manifest.

Analyzing GoogleUtilitiesComponents ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing MLImage ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing MLKitBarcodeScanning ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/MLKitBarcodeScanning/Frameworks/MLKitBarcodeScanning.framework/PrivacyInfo.xcprivacy
API usage analysis result(s): 2
[0] NSPrivacyAccessedAPICategoryFileTimestamp:stat,fstat,lstat:./Pods/MLKitBarcodeScanning/Frameworks/MLKitBarcodeScanning.framework/MLKitBarcodeScanning
[1] NSPrivacyAccessedAPICategorySystemBootTime:systemUptime:./Pods/MLKitBarcodeScanning/Frameworks/MLKitBarcodeScanning.framework/MLKitBarcodeScanning
✅ All required API reasons have been described in the privacy manifest.

Analyzing MLKitCommon ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/MLKitCommon/Frameworks/MLKitCommon.framework/PrivacyInfo.xcprivacy
API usage analysis result(s): 4
[0] NSPrivacyAccessedAPICategoryFileTimestamp:stat,NSURLContentModificationDateKey:./Pods/MLKitCommon/Frameworks/MLKitCommon.framework/MLKitCommon
[1] NSPrivacyAccessedAPICategorySystemBootTime:systemUptime:./Pods/MLKitCommon/Frameworks/MLKitCommon.framework/MLKitCommon
[2] NSPrivacyAccessedAPICategoryDiskSpace:NSFileSystemFreeSize:./Pods/MLKitCommon/Frameworks/MLKitCommon.framework/MLKitCommon
[3] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/MLKitCommon/Frameworks/MLKitCommon.framework/MLKitCommon
✅ All required API reasons have been described in the privacy manifest.

Analyzing MLKitVision ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing OrderedSet 🎯 ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing PromisesObjC ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/PromisesObjC/Sources/FBLPromises/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing PromisesSwift ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/PromisesSwift/Sources/Promises/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing SwiftProtobuf ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/SwiftProtobuf/Sources/SwiftProtobuf/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing nanopb 🎯 ...
💡 Found privacy manifest file(s): 1
[0] ./Pods/nanopb/spm_resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.


==================== Analyzing Flutter Plugins Directory ====================

Analyzing app_settings-5.1.1 ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing contentsquare-3.10.0 ...
💡 Found privacy manifest file(s): 1
[0] /Users/fstonge/.pub-cache/hosted/pub.dev/contentsquare-3.10.0/ios/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing firebase_analytics-10.10.4 ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing firebase_core-2.30.1 ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing firebase_crashlytics-3.5.4 ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing firebase_performance-0.9.4+4 ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing firebase_remote_config-4.4.4 ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing flutter_keyboard_visibility-6.0.0 ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing flutter_native_splash-2.4.0 ...
💡 Found privacy manifest file(s): 1
[0] /Users/fstonge/.pub-cache/hosted/pub.dev/flutter_native_splash-2.4.0/ios/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing flutter_secure_storage-9.0.0 ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing geolocator_apple-2.3.7 🎯 ...
💡 Found privacy manifest file(s): 1
[0] /Users/fstonge/.pub-cache/hosted/pub.dev/geolocator_apple-2.3.7/ios/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing haptic_feedback-0.4.2 ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing local_auth_darwin-1.2.2 ...
💡 Found privacy manifest file(s): 1
[0] /Users/fstonge/.pub-cache/hosted/pub.dev/local_auth_darwin-1.2.2/darwin/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing local_auth_ios-1.1.7 ...
💡 Found privacy manifest file(s): 1
[0] /Users/fstonge/.pub-cache/hosted/pub.dev/local_auth_ios-1.1.7/ios/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing mobile_scanner-5.0.1 ...
💡 Found privacy manifest file(s): 1
[0] /Users/fstonge/.pub-cache/hosted/pub.dev/mobile_scanner-5.0.1/ios/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing open_file-3.3.2 ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0

Analyzing package_info_plus-8.0.0 🎯 ...
💡 Found privacy manifest file(s): 1
[0] /Users/fstonge/.pub-cache/hosted/pub.dev/package_info_plus-8.0.0/ios/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing path_provider_foundation-2.3.2 ...
💡 Found privacy manifest file(s): 1
[0] /Users/fstonge/.pub-cache/hosted/pub.dev/path_provider_foundation-2.3.2/darwin/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing sqflite-2.3.3 🎯 ...
💡 Found privacy manifest file(s): 1
[0] /Users/fstonge/.pub-cache/hosted/pub.dev/sqflite-2.3.3/darwin/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing url_launcher_ios-6.2.5 🎯 ...
💡 Found privacy manifest file(s): 1
[0] /Users/fstonge/.pub-cache/hosted/pub.dev/url_launcher_ios-6.2.5/ios/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing video_player_avfoundation-2.5.7 🎯 ...
💡 Found privacy manifest file(s): 1
[0] /Users/fstonge/.pub-cache/hosted/pub.dev/video_player_avfoundation-2.5.7/darwin/Resources/PrivacyInfo.xcprivacy
API usage analysis result(s): 0
✅ All required API reasons have been described in the privacy manifest.

Analyzing webview_flutter_wkwebview-2.9.5 🎯 ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 0


==================== Analysis completed! ⏰: 312s 💡: 31 ⚠️ : 26 🛠️ : 5 ✅: 29 🎯: 21 ====================

⚠️ 🛠️  https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api
🎯 https://developer.apple.com/support/third-party-SDK-requirements

🔔 If the directory you are analyzing is the app project directory, your app's privacy manifest may be affected by these analysis results: 4
[0] NSPrivacyAccessedAPICategoryFileTimestamp:stat,NSURLContentModificationDateKey:./Pods/MLKitCommon/Frameworks/MLKitCommon.framework/MLKitCommon
[1] NSPrivacyAccessedAPICategorySystemBootTime:systemUptime:./Pods/MLKitCommon/Frameworks/MLKitCommon.framework/MLKitCommon
[2] NSPrivacyAccessedAPICategoryDiskSpace:NSFileSystemFreeSize:./Pods/MLKitCommon/Frameworks/MLKitCommon.framework/MLKitCommon
[3] NSPrivacyAccessedAPICategoryUserDefaults:NSUserDefaults:./Pods/MLKitCommon/Frameworks/MLKitCommon.framework/MLKitCommon

🌟 If you found this script helpful, please consider giving it a star on GitHub. Your support is appreciated. Thank you!
🔗 Homepage: https://github.com/crasowas/app_store_required_privacy_manifest_analyser
🐛 Report issues: https://github.com/crasowas/app_store_required_privacy_manifest_analyser/issues

    ~/Dev/Co/spark-mobileapp-shop/ios    master !12 ?1                                                                    ✔  5m 12s   07:44:04 AM  
```

**Comment by crasowas** [comment]
Thank you for your feedback! However, the analysis results meet our expectations.

We can use the `file` command to ascertain that the binary file within `MLKitCommon.framework` is of the file type `current ar archive`, implying that it is a statically linked library.

Statically linked libraries are fully copied into the executable file of the application bundle during compilation. Therefore, during the App Store review process, they are considered part of your app's code. This is why the results of `MLKitCommon` analysis could affect your app’s privacy manifest.
